### PR TITLE
Docs/general update nov2023

### PR DESCRIPTION
**Pull request by blankdots**
### Describe the pull request:

- [x] Bug fix
- [x] Documentation change
- [x] Build change 

### Changes made:

- update links for `sda-*` repositories 
- update dictionary (sort lines ascending, delete duplicates, transform to lowercase)
- try to introduce again auto-merge documentation updates (:crossed_fingers:  this time i got the right set of conditions)
- remove images not utilised
- update main architecture image to reflect storage use, preview:
image

### CODE REVIEW

**Review by kjellp — APPROVED** [review]
LGTM overall. Was a bit unsure about the "sensitive-data-archive/tree/main/<component>" paths?  And the use of tags to separate/run different microsevices from the same overall image (thought there will be a few different images, and a specific cmd/servicename to be provided in combination with the right image).

**Comment by blankdots** [comment]
> Was a bit unsure about the "sensitive-data-archive/tree/main/" paths?

not sure how else to reference those, but we can fix in the next iteration 

> And the use of tags to separate/run different microsevices from the same overall image

we can bring that up in our bi-weekly, however that is something already existing and i just used that

File: README.md [hunk 1]
```diff
@@ -11,30 +11,30 @@ SDA is divided into several components, which can be deployed either for Federat
 
 ### Core Components
 
-Source code for core components (unless specified otherwise) is available at: https://github.com/neicnordic/sda-pipeline 
+Source code for core components is available at: https://github.com/neicnordic/sensitive-data-archive
 
 | Component     | Role |
 |---------------|------|
-| inbox         | SFTP, S3 or HTTPS server, acting as a dropbox, where user credentials are fetched from CentralEGA or via ELIXIR AAI. https://github.com/neicnordic/sda-s3proxy/ or https://github.com/neicnordic/sda-inbox-sftp |
+| inbox         | SFTP, S3 or HTTPS server, acting as a dropbox, where user credentials are fetched from CentralEGA or via LifeScience AAI. [s3inbox](https://github.com/neicnordic/sensitive-data-archive/tree/main/sda/cmd/s3inbox/s3inbox.md) or [sftp-inbox](https://github.com/neicnordic/sensitive-data-archive/tree/main/sda-sftp-inbox) |
```
Comment (jbygdell) [inline README.md:18]:
```suggestion
| inbox         | SFTP, S3 or HTTPS server, acting as a dropbox, where user credentials are fetched from CentralEGA or via LifeScience AAI. [s3inbox](https://github.com/neicnordic/sensitive-data-archive/tree/main/sda/cmd/s3inbox/s3inbox.md) or [sftp-inbox](https://github.com/neicnordic/sensitive-data-archive/tree/main/sda-sftp-inbox/README.md) |
```

File: README.md [hunk 2]
```diff
@@ -11,30 +11,30 @@ SDA is divided into several components, which can be deployed either for Federat
...
 | intercept     | The intercept service relays message between the queue provided from the federated service and local queues. **(Required for Federated EGA use case)** |
 | ingest        | Split the Crypt4GH header and move the remainder to the storage backend. No cryptographic task, nor access to the decryption keys. |
 | verify        | Decrypt the stored files and checksum them against their embedded checksum. |
 | archive       | Storage backend: as a regular file system or as a S3 object store. |
 | finalize      | Handle the so-called _Accession ID_ to filename mappings from CentralEGA. |
 | mapper        | The mapper service register mapping of accessionIDs (IDs for files) to datasetIDs. |
 | data out API  | Provides a download/data access API for streaming archived data either in encrypted or decrypted format - source at: https://github.com/neicnordic/sda-doa |
+| download      | Provides a download/data access API for streaming (decrypted) archived data - source at: https://github.com/neicnordic/sda-download |
```
Comment (jbygdell) [inline README.md:26]:
```suggestion
| download      | Provides a download/data access API for streaming (decrypted) archived data - source at: [https://github.com/neicnordic/sensitive-data-archive](https://github.com/neicnordic/sensitive-data-archive/tree/main/sda-download/README.md) |
```

File: docs/deploy.md [hunk 3]
```diff
@@ -9,22 +9,13 @@ Swarm](https://docs.docker.com/engine/swarm/) for production.
...
-    (based on Apache Mina, container base
-    *openjdk:13-alpine*);
--   `neicnordic/sda-doa`, provides the data out service (Data Out API);
--   `neicnordic/sda-s3-proxy`, provides the inbox service via a s3 proxy
-    (S3 proxy inbox, minimal container with static binary and support
-    files).
+- `ghcr.io/neicnordic/sensitive-data-archive:<version>-postgres` - PostgreSQL database
+- `ghcr.io/neicnordic/sensitive-data-archive:<version>-rabbitmq` - RabbitMQ message broker
+- `ghcr.io/neicnordic/sensitive-data-archive:<version>-sftp-inbox` - sftp inbox
+- `ghcr.io/neicnordic/sensitive-data-archive:<version>-<service>` - where `<service>` corresponds to services such as: `finalize`, `ingest`, `intercept`, `verify`, `mapper`, `download`, `s3inbox`, `auth`
```
Comment (jbygdell) [inline docs/deploy.md:21]:
```suggestion
- `ghcr.io/neicnordic/sensitive-data-archive:<version>-auth` - authentication service
- `ghcr.io/neicnordic/sensitive-data-archive:<version>-download` - download service
- `ghcr.io/neicnordic/sensitive-data-archive:<version>` - all other services such as: `finalize`, `ingest`, `intercept`, `verify`, `mapper` and `s3inbox`
```

**Review by jbygdell — CHANGES_REQUESTED** [review]

**Review by jbygdell — APPROVED** [review]

File: docs/connection.md [hunk 4]
```diff
@@ -275,7 +272,7 @@ the `completed` queue.
 }
 ```
 
-The message sent from the sda-pipeline `finalize` service to the `backup` service via `completed` queue.
+The message sent from the `finalize` to the `completed` queue.
```
Comment (aaperis) [inline docs/connection.md:275]:
```suggestion
The message sent from the `finalize` service to the `completed` queue.
```

File: docs/deploy.md [hunk 5]
```diff
@@ -9,22 +9,15 @@ Swarm](https://docs.docker.com/engine/swarm/) for production.
 
 The production deployment repositories are:
 
--   [Kubernetes Helm charts](https://github.com/neicnordic/sda-helm/);
+-   [Kubernetes Helm charts](https://github.com/neicnordic/sensitive-data-archive/tree/main/charts);
 -   [Docker Swarm
     deployment](https://github.com/neicnordic/LocalEGA-deploy-swarm/).
 
-The following container images are used in the deployments:
+The following container image is used in the deployments `neicnordic/sensitive-data-archive`, provides the SDA services as well as PostgreSQL and RabbitMQ configuration. The tag is what separates the services:
```
Comment (aaperis) [inline docs/deploy.md:16]:
```suggestion
`neicnordic/sensitive-data-archive`, provides the SDA services as well as PostgreSQL and RabbitMQ configuration. The following container image is used in the deployments where the tag separates between services:
```

**Review by aaperis — APPROVED** [review]
Looks good! small suggestions to (possibly) improve clarity :-)

**Pull request merged**